invesalius/data/coregistration.py

```diff
@@ -102,7 +106,7 @@ def tracker_to_image(m_change, m_probe_ref, r_obj_img, m_obj_raw, s0_dyn):
     return m_img
 
 
-def image_to_tracker(m_change, target, icp):
+def image_to_tracker(m_change, coord_raw, target, icp, obj_data):
```

Comment:
Would be really great to give more details in the comments in this method, what each step is doing. This will help to remember easier in the future.

Reply:
Done with commit #[356a241](https://github.com/invesalius/invesalius3/pull/472/commits/356a2414b5e3fa63c508b96aacc08807e9c19328)